### DISCUSSION TITLE
floating elements "ticker" scroll speed

### DISCUSSION THREAD

**Discussion by Ma-Tchelo** [elementor/elementor#28806] [Feature Request]
### Prerequisites

- [X] I have searched for similar feature requests in both open and closed discussions and cannot find an existing request.
- [X] I have verified that the feature is still missing from the latest stable versions of Elementor or Elementor Pro.

### What problem is your feature request going to solve? Please describe..

I made a floating element in the footer where the "scroll speed" option of the "ticker" has a different problem on desktop and mobile phones: to get good speed on the phone, it is extremely fast on the desktop.... to get good speed on the desktop is extremely slow on cell phones.

![elementor-2](https://github.com/user-attachments/assets/93563b6b-2069-4c3b-85b3-5ef502b1e864)

### Describe the solution you'd like.

have different speeds on desktop and mobile

### Agreement

- [X] I agree that my request may be closed without action if it doesn't meet all the requirements.

**Comment by bperry420** [comment]
This problem makes the Ticker Floating Bar unusable. I agree - to get good speed on the phone, it is extremely fast on the desktop.... to get good speed on the desktop is extremely slow on cell phones.

**Comment by errorlevel101** [comment]
While the speed is OK on the desktop, the ticker runs extremely slowly. The speed must be adjustable separately.
